pom.xml

```diff
@@ -23,7 +23,10 @@
   <groupId>io.cdap.cdap</groupId>
   <artifactId>cdap</artifactId>
   <version>6.9.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
+    <modules>
+        <module>cdap-relational-spi</module>
```

Comment:
Does this belong in the existing default profile?

Reply:
If it is shipped by default, I think both the SPI and calcite extension can be added to modules list in default profile.